Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -4941,10 +4938,6 @@ static bool selectionIsInsideFixedPositionContainer(LocalFrame& frame)
     if (!document)
         return String();
 
-    if (osNameForUserAgent() == "iPhone OS"_s) {
-        if (document->quirks().shouldAvoidUsingIOS13ForGmail())
-            return standardUserAgentWithApplicationName({ }, "12_1_3"_s);
-    }
```

Comment:
@brentfulgham For the reviewer, the initial patch was adding more code than what I removed here. Not sure if I should revert everything as it was initially. 
https://github.com/WebKit/WebKit/commit/b49d99266d0d84c0331d69f63263f5a856a5e773

Reply:
I don't see any cases where this method returns anything *but* `String()`. I think you should simplify to match `WebPageMac.mm` and just return `String()`. Extra credit: Move the method in `WebPageMac.mm` to `WebPageCocoa.mm` and delete this entirely.

Reply:
This is declared in `WebPage.h`
https://github.com/WebKit/WebKit/blob/c2e4b673742acda4d321dbfa88437ae52fcda2ab/Source/WebKit/WebProcess/WebPage/WebPage.h#L584

Then used in `WebPagePlayStation.cpp`

https://github.com/WebKit/WebKit/blob/c2e4b673742acda4d321dbfa88437ae52fcda2ab/Source/WebKit/WebProcess/WebPage/playstation/WebPagePlayStation.cpp#L56-L62

and `WebPageWin.cpp`

https://github.com/WebKit/WebKit/blob/c2e4b673742acda4d321dbfa88437ae52fcda2ab/Source/WebKit/WebProcess/WebPage/win/WebPageWin.cpp#L77-L80

and `WebPageGLib.cpp`

https://github.com/WebKit/WebKit/blob/c2e4b673742acda4d321dbfa88437ae52fcda2ab/Source/WebKit/WebProcess/WebPage/glib/WebPageGLib.cpp#L201-L207

and `WebPageIOS.mm`

https://github.com/WebKit/WebKit/blob/c2e4b673742acda4d321dbfa88437ae52fcda2ab/Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm#L4929-L4949

So here if I just return `String()` like others that seems to work without issues. 
There's a strange quirk which is described in rdar://116201535



and `WebPageMac.mm`

https://github.com/WebKit/WebKit/blob/c2e4b673742acda4d321dbfa88437ae52fcda2ab/Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm#L799-L802

Reply:
the FIXME was added by @achristensen07 
in https://github.com/WebKit/WebKit/commit/57435dd4c7a3967e2fa9eb4da36183dc061d7ea5

and the code was added by @achristensen07 
https://github.com/WebKit/WebKit/commit/93cc4767ec04fbe1027cda51ccb2ad1d47fa879f

in the context of site isolation
https://github.com/WebKit/WebKit/commit/93cc4767ec04fbe1027cda51ccb2ad1d47fa879f#diff-49068a67a1cd1b7b267adbee4e47477a00787e24f9cfcbd6df2372be36b5778aL4467-R4479

but I do not see a rationale for it.

Reply:
@achristensen07 reached out privately and said that you should ignore that FIXME.

Reply:
@karlcow I think you should finish this patch using the simple `return String()` (or `return { }`) so that iOS and macOS match. In a follow-up patch, I think we might want to move to the form Playstation and Glib use so that we can fully move to the UA Quirks living in WebKit (instead of at the browser layer). That would allow us to collapse all platforms to a single implementation at the `WebPage` level, with the quirks living in the `WebCore::standardUserAgentForURL(...)` method.

Reply:
Good idea! Thanks. This is resolved.